apps/SageAccountWeb/src/components/AccountSettings.tsx

```diff
@@ -101,9 +101,11 @@ export const AccountSettings = () => {
     const current = new Date()
     const nextYear = new Date()
     nextYear.setFullYear(current.getFullYear() + 1)
+    const hostname = window.location.hostname.toLowerCase()
     cookies.set(SynapseConstants.DATETIME_UTC_COOKIE_KEY, isUTCTime, {
       path: '/',
       expires: nextYear,
+      domain: hostname.endsWith('.synapse.org') ? '.synapse.org' : undefined,
```

Comment:
[MDN](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Set-Cookie#domaindomain-value)
> Contrary to earlier specifications, leading dots in domain names (.example.com) are ignored.
>
> Multiple host/domain values are not allowed, but if a domain is specified, then subdomains are always included.

```suggestion
      domain: hostname.endsWith('.synapse.org') ? 'synapse.org' : undefined,
```